src/contained.js

```diff
@@ -0,0 +1,27 @@
+import { flip, contains } from 'ramda';
+
+/**
+ * Returns true if the specified value is equal, in R.equals terms,
+ * to at least one element of the given list; false otherwise.
+ *
+ * Like {@link http://ramdajs.com/docs/#contains|R.contains} but with argument order reversed.
+ *
+ * @func contained
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.7.0|v2.7.0}
+ * @category List
+ * @sig [a] → a → Boolean
+ * @param {Array} list The array to consider
```

Comment:
'list' instead of 'array'

Also, if it supports strings, this should be `{Array|String} I think.